src/internal/i18n/i18n.ts

```diff
@@ -1,13 +0,0 @@
-import { useTranslation as useReactI18NextTranslation } from 'react-i18next';
-
-// IMPORTANT: This file adds comments recognized by the react-i18next-parser so that
-// labels declared in console-extensions.json are added to the message catalog.
-
-// t('plugin__forklift-console-plugin~Virtualization')
-// t('plugin__forklift-console-plugin~Providers for VM Import')
-// t('plugin__forklift-console-plugin~Plans for VM Import')
-// t('plugin__forklift-console-plugin~Mappings for VM Import')
-
-export function useTranslation() {
```

Comment:
can we remove the comments (some time) after #82 ?  removing the comments results in losing the translations and then we are re-adding them

Reply:
deleted :+1: